library/build.gradle

```diff
@@ -1,18 +1,21 @@
 apply plugin: 'com.android.library'
 
-dependencies {
-    implementation files('../../shared/framework_all.jar')
-    implementation 'com.klinkerapps:logger:1.0.3'
-    implementation 'com.squareup.okhttp:okhttp:2.5.0'
-    implementation 'com.squareup.okhttp:okhttp-urlconnection:2.5.0'
-}
-
 android {
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    flavorDimensions "version"
+    productFlavors {
+      "lp2" {
+        dimension "version"
+      }
+      "lp3" {
+        dimension "version"
+      }
+    }
```

Comment:
Low and behold, an android library can have flavors too!

Reply:
Spicy!